test/model_test.dart

```diff
@@ -731,18 +731,18 @@ void main() {
     // are exported out through one library
     test('ExtendingClass has a super class that is also in the same library',
         () {
-      expect(ExtendingClass.supertype.name, equals('BaseClass'));
-      expect(
-          ExtendingClass.supertype.element.library.name, equals('two_exports'));
+      expect(ExtendingClass.superChain[0].name, equals('BaseClass'));
```

Comment:
I tend to use `foo.first` instead of `foo[0]` in many cases - it's a bit more semantic; YMMV.